test/lib/eslint/BaseEslintEngine.test.ts

```diff
@@ -68,7 +68,7 @@ describe('Tests for BaseEslintEngine', () => {
 				Mockito.verify(StaticDependenciesMock.resolveTargetPath(target.target)).called();
 
 				// verify config
-				const capturedConfig = Mockito.capture(StaticDependenciesMock.createCLIEngine).first();
+				const capturedConfig = Mockito.capture(StaticDependenciesMock.createCLIEngine).second();
```

Comment:
This changed because createCLIEngine is called twice in the test because #getCatalog is called from the #run command.